probe-rs/src/architecture/arm/core/armv8a.rs

```diff
@@ -1215,6 +1227,14 @@ impl<'probe> MemoryInterface for Armv8a<'probe> {
         Ok(())
     }
 
+    fn read_16(&mut self, address: u64, data: &mut [u16]) -> Result<(), Error> {
+        for (i, word) in data.iter_mut().enumerate() {
+            *word = self.read_word_16(address + ((i as u64) * 4))?;
```

Comment:
Maybe I'm going to ask something obvious, but what is 4 here?

Reply:
fixed